torchvision/models/swin_transformer.py

```diff
@@ -357,7 +374,7 @@ def __init__(
         # build SwinTransformer blocks
         for i_stage in range(len(depths)):
             stage: List[nn.Module] = []
-            dim = embed_dim * 2 ** i_stage
+            dim = embed_dim * 2**i_stage
```

Comment:
Revert, that's likely to break the linter.

Reply:
hmm... this is strange, somehow my local ufmt is the one changing this line. Let me fix my local ufmt and revert this.